newIDE/app/src/UI/IconButton.js

```diff
@@ -30,14 +30,17 @@ type Props = {|
   onContextMenu?: () => void,
   disabled?: boolean,
   edge?: 'start' | 'end' | false,
+  id?: string,
 
   style?: {|
     padding?: number,
     width?: number,
     height?: number,
+    cursor?: 'pointer',
```

Comment:
I think that we mentioned at some point that we didn't want any pointer in the app, to avoid being to "web".
Instead we'd rely on hover effects being visible enough.
I don't know if this rule still stands. But is the cursor really needed?

Reply:
You're right. Well it's for the buttons in the tooltip that is "outside of the theme". But I don't have hover effects on those buttons. Might not be worth allowing the cursor to be set for icon buttons "just" for that